### ISSUE TITLE
Add Prisma support to Prettier

### ISSUE THREAD

**Issue by schickling** [prisma/prisma#703]
It would be great if once the Prisma language + conventions have settled to add official Prisma support to Prettier (similar to other supported tools like styled-components or GraphQL).

**Comment by ajotaos** [comment]
This should definitely be part of the support for prettier. It's hard to follow the format if not using VSCode.

**Comment by pantharshit00** [comment]
Closing this one in favor of https://github.com/prisma/prisma/issues/1761 as that one has more up votes and these two issues are duplicates.
